### DISCUSSION TITLE
sentinel熔断日志如何不打印到文件中？貌似没有相关配置

### DISCUSSION THREAD

**Discussion by WTY1997** [alibaba/Sentinel#2888] [Q&A]
如题

**Comment by su-yh** [comment]
`${appName}-metrics.log.pid${pid}.yyyy-MM-dd.[number]`  对于这个日志文件，你可以通过配置项："csp.sentinel.metric.flush.interval" 来控制，只要值小于等于 0 就可以关闭了。具体的位置你可以看一下：`com.alibaba.csp.sentinel.slots.block.flow.FlowRuleManager#startMetricTimerListener()`

**Reply by Dynasty-develop** [reply]
command-center.log和sentinel-record.log有办法关闭输出不
